Microsoft.DotNet.Interactive/Commands/RestoreNugetDirective.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Interactive.Commands
+{
+    public class RestoreNugetDirective : KernelCommandBase
```

Comment:
>RestoreNugetDirective [](start = 17, length = 21)

Would RestorePackages capture the intent here? You've removed "Nuget" from other class names, and "Directive" sounds like the direct object here, which isn't accurate.